### DISCUSSION TITLE
cannot open source file "ace/Trace.h"

### DISCUSSION THREAD

**Discussion by Oles7** [OpenDDS/OpenDDS#4484] [Q&A]
Unable to build  DDS_TAOv2.sln because it cannot open ace source files

**Comment by mitza-oci** [comment]
Set the ACE_ROOT environment variable.
See https://opendds.readthedocs.io/en/latest-release/devguide/quickstart/windows.html

**Comment by Oles7** [comment]
cannot open source file "dds/DdsDcpsCoreC.h""

I have DDS_ROOT set to OpenDDS-3.27\dds

**Comment by Oles7** [comment]
Also tried setting it to OpenDDS-3.27 still not working

**Comment by mitza-oci** [comment]
If you follow all the steps in https://opendds.readthedocs.io/en/latest-release/devguide/quickstart/windows.html then you don't need to set environment variables manually

**Comment by Oles7** [comment]
I did. I run configure, then  run the devenv command with the solution it tells me to from the command line in VS

**Comment by jrw972** [comment]
Please attach a text file containing a transcript of your terminal session starting with `configure` and ending with `DDS_TAOv2_all.sln`.
